File: Genelife.Web/DTOs/Human.cs
using GeneLife.Core.Components.Characters;

namespace Genelife.Web.DTOs
{
    public class Human
    {
        public string Identity { get; set; }
        public HumanStats Stats { get; set; }
        public string Wallet { get; set; }
        public string[] Objectives { get; set; }
        public string Position { get; set; }
    }
}

File: Genelife.Web/ClientApp/src/models/Human.ts
export class HumanStats {
    public stamina: string = ""
    public thirst: string = ""
    public hunger: string = ""
    public damage: string = ""
}

export class Human {
    wallet: string = ""
    identity: string = ""
    stats?: HumanStats
    objectives: string[] = []
}